R/miscFunctions.R

```diff
@@ -472,3 +472,23 @@ getGenesetNamesFromCollection <- function(inSCE, geneSetCollectionName) {
                      code)
   }
 }
+
+
+renameClusters <- function(inSCE, clusterName, from, to, newClusterName = NULL) {
```

Comment:
Thanks @nathanpalamuttam for this work! Could you also add documentation to this function using roxygen skeleton?

Reply:
Also, you need to directly specify the namespace of the mapvalues function:

plry::mapvalues

Lastly, can you close this PR and open one that merges into the 'devel' branch. thanks